### PR TITLE
test: [M3-8435] - Cypress integration test for Object Storage Gen2: E3 Endpoint

### PR DESCRIPTION
**Pull request by jaalah-akamai**
## Changes  🔄
- New Integration Test for Object Storage Gen2: E3 Endpoint - Bucket Creation

## Target release date 🗓️
N/A

## How to test 🧪
- `yarn cy:run -s "cypress/e2e/core/objectStorageGen2/bucket-create-gen2.spec.ts"`

Requirements:
- Visiting /object-storage/buckets/create causes "Create Bucket" drawer to open
- Users can select "E3" endpoint type for regions that support E3 endpoint types
    -  This will require mocking the response to the API endpoint types request to ensure that "E3" is available for the selected region
- Upon selecting "E3" endpoint type, rate limit table appears
    -  Rate limit table contains expected data
        -  (This data is currently pending, but notably it should differ from E2's rate limiting info)
- Informational notice text is displayed alongside rate limit table:
    -  Specifies the maximum Requests Per Second (RPS) for a bucket. To increase it to High, open a support ticket. Understand bucket rate limits.
- Upon clicking "Create Bucket" with "E3" endpoint type selected, Cloud fires an API bucket create request with the following data in its payload:
    -  cors_enabled is false
    -  endpoint_type is 'E3'
    -  s3_endpoint is present if corresponding endpoint mock contains non-null s3_endpoint
        -  At least one of these subtasks should test the case when s3_endpoint is specified, and at least one of these subtasks should test the case when s3_endpoint is omitted
-  Upon creating bucket, the "Create Bucket" drawer closes and new bucket appears in the landing page
    -  The correct information is displayed alongside the new bucket:
        -  Label
        -  Region
        -  Endpoint type
        -  Endpoint URL

## As an Author I have considered 🤔
*Check all that apply*

- [x] 👀 Doing a self review
- [x] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a [changeset](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md#writing-a-changeset)
- [x] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

**Review by AzureLatte — APPROVED** [review]
Flaky issue in CI tests.
The newly added test passed locally though. 
Thank you!

File: packages/manager/cypress/e2e/core/objectStorageGen2/bucket-create-gen2.spec.ts [hunk 1]
```diff
@@ -197,4 +197,195 @@ describe('Object Storage Gen2 create bucket tests', () => {
...
+            'OTHER',
+          ];
+          const expectedBasicValues = [
+            'Basic',
+            '2,000',
+            '500',
+            '100',
+            '200',
+            '400',
+          ];
```
Comment (coliu-akamai) [inline packages/manager/cypress/e2e/core/objectStorageGen2/bucket-create-gen2.spec.ts:297]: same with these two rows to reduce repetition

**Comment by jaalah-akamai** [comment]
Thanks I'll make the changes in this PR 🙏

**Comment by jaalah-akamai** [comment]
I'm not sure how the `// Confirm that bucket is created, initiate deletion for cleanup` is passing in the CI because locally it always fails, but this was code that was pre-existing since E0

**Pull request merged**